setup.py

```diff
@@ -17,6 +17,7 @@
 
 requirements = [
     "numpy>=1.17",
+    "numpy<1.22",
```

Comment:
This isn't the best way to do this. This is basically equivalent to doing `pip install numpy>=1.17 numpy<1.22`, you can just use a comma to put this on the same line 

```suggestion
    "numpy>=1.17,<1.22",
```